0.5/index.bs

```diff
@@ -652,6 +652,15 @@ Version History {#history}
   </tr>
 </table>
 
+Changelog {#changelog}
+======================
+
+Version 0.6
```

Comment:
I understand this to be a collection of more fine-grained changes between versions. If so, can we move it to the top-level? This could lead to confusion as to the state of "0.6" if in the 0.5 spec which could lead to confusion.

Reply:
So in my head 0.5 is released and therefore fixed, because it is not versioned at a more granular level (e.g., 0.5.0, 0.5.1, 0.5.2). See my reasoning for why I think 0.x should be fixed on release in https://github.com/ome/ngff/pull/278#issuecomment-2493728701.

> If so, can we move it to the top-level?

I don't see the harm (and I think it makes it easier to understand) in calling out which version a change happened in (and coversely, what list of changes happened in a given version)

This all ties back to https://github.com/ome/ngff/issues/276 it seems - shouldn't there be an "active development" version of the spec, which changes are made against, which will become the version after the most recent versioned release?

Reply:
Hmm.... to be clear: did you edit in `/latest/` or in `/0.5/`? @normanrz's PR may have moved your target.

Reply:
Latest is currently symlinked to /0.5/: https://github.com/ome/ngff/commit/72c443041f49c0ab4c414d5d40f196143c612fd1, so I don't think there is a latest copy at the moment?

Reply:
Nope. That was what I was trying to figure out. If you had made your change on `/latest/` then @normanrz's change automagically integrated it.